src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -341,57 +277,32 @@ export async function handlePinListItem(
   }
 }
 
-export async function listItemDeleteController(
-  req: Request,
-  res: Response
-): Promise<void> {
+export async function listItemDeleteController(req: Request, res: Response): Promise<void> {
   const { listId, listItemId } = req.params;
   const userId = req?.user?.userData?.id;
```

Comment:
Needs a check that the user can access the list and cast to a number prior to querying the DB with those ids.